spikes/data/loderunner/app/WebValidation/Main.cs

```diff
@@ -602,6 +591,7 @@ private void LogToConsole(Request request, ValidationResult valid, PerfLog perfL
                     { "Duration", Math.Round(perfLog.Duration, 2) },
                     { "ContentLength", perfLog.ContentLength },
                     { "CVector", perfLog.CorrelationVector },
+                    { "CVectorBase", perfLog.CorrelationVectorBase },
                     { "Tag", perfLog.Tag },
```

Comment:
Should we log CVectorBase in the app log as well?

Reply:
yes - there's a task on the board to log in ngsa and DS. Will try to get into this PR but may be the next PR.